uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1017.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1017 {
+
+  /**
+   * Migrating Phone Types and Deleting unwanted Phone Types
+   */
+  public function upgrade_1017() {
+    $this->migratePhoneTypes();
+    $this->relabelPhoneType("phone", "Landline");
```

Comment:
Please use single quotes for the strings and also for other strings in this function.
Also I did a quick check in my local civicrm installation and the Phone label starts with a capital letter, does this differ per site or something?

Reply:
The case is irrelevant, but i will change it